src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void done(String line, Task[] taskList) {
+        String taskNumber = line.substring(5);
+        int taskListElement = Integer.parseInt(taskNumber) - 1;
+        taskList[taskListElement].markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("[X] " + taskList[taskListElement].getDescription());
+    }
+
+    private static void list(Task[] taskList, int listCount) {
+        int taskCount = 1;
+        for (int i = 0; i < listCount; i++) {
+            System.out.println(taskCount + "." + taskList[i]);
+            taskCount++;
+        }
+    }
+
+    private static Todo todo(String[] stringList, Task[] taskList, int listCount, String line) {
+        char taskType = line.toUpperCase().charAt(0);
+        String taskDisplay = line.substring(5);
+
+        stringList[listCount] = taskDisplay;
+        Todo todoTask = new Todo(taskDisplay, taskType);
+        taskList[listCount] = todoTask;
+        return todoTask;
+    }
+
+    private static Deadline deadline(String[] stringList, Task[] taskList, int listCount, String line) {
+        int startingIndex = line.indexOf("/");
+        String taskDisplay = line.substring(9, startingIndex);
+        String doBy = "(" + line.substring(startingIndex + 1) + ")";
+        char taskType = line.toUpperCase().charAt(0);
+
+        stringList[listCount] = taskDisplay;
+        Deadline deadlineTask = new Deadline(taskDisplay, taskType, doBy);
+        taskList[listCount] = deadlineTask;
+        return deadlineTask;
+    }
+
+    private static Event event(String[] stringList, Task[] taskList, int listCount, String line) {
+        int startingIndex = line.indexOf("/");
+        String taskDisplay = line.substring(6, startingIndex);
+        String doBy = "(" + line.substring(startingIndex + 1) + ")";
+        char taskType = line.toUpperCase().charAt(0);
+
+        stringList[listCount] = taskDisplay;
+        Event eventTask = new Event(taskDisplay, taskType, doBy);
+        taskList[listCount] = eventTask;
+        return eventTask;
+    }
+
+    private static void printTask(Task task, int listCount) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + listCount + " tasks in the list.");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
```

Comment:
Could consider putting this in a separate greetUser() function for ease of update later on